profiles/api/webhook.py

```diff
@@ -11,6 +12,8 @@
 from profiles.orcid import OrcidClient
 from profiles.repositories import OrcidTokens, Profiles
 
+LOGGER = logging.getLogger()
```

Comment:
should use `getLogger(__name__)`? Same for the other calls introduced, but I'm not sure what the rest of the application is doing

Reply:
I remember you said that I should use the root logger on another PR. I'll update these to use the module logger.